statsmodels/sandbox/gam_gsoc2015/gam_example.R

```diff
@@ -0,0 +1,47 @@
+## This files contains the R code to generate some test for the GAM function ##
+## Documentation for MGCV is available at: http://cran.r-project.org/web/packages/mgcv/mgcv.pdf
+
+library('pracma')
+library('mgcv')
+#library('gam')
+n = 100
+x = seq(from = -1, to = 1, length.out = n)
+
+funz = function(x){
+  2 * x * x * x - x + rnorm(n = n, mean = 0, sd = .1)
```

Comment:
You need to set a seed so you can replicate. (I don't see any.) Otherwise you will get new numbers on each run, and we won't be able to redo.

Also random numbers are not the same between R and Python/numpy, so you need to save them to a csv file for reloading across packages.